jsk_interactive_markers/jsk_interactive_marker/src/polygon_marker.cpp

```diff
@@ -59,7 +59,6 @@ void publishClickedPolygon(jsk_recognition_msgs::Int32Stamped& msg)
 void processFeedback(
   const visualization_msgs::InteractiveMarkerFeedbackConstPtr &feedback)
 {
-  boost::mutex::scoped_lock(mutex);
```

Comment:
same as `bounding_box_marker`.
do we still need boost::mutex mutex; in line 45?